src/core/ext/filters/client_channel/xds/xds_client_stats.h

```diff
@@ -20,13 +20,12 @@
 #define GRPC_CORE_EXT_FILTERS_CLIENT_CHANNEL_XDS_XDS_CLIENT_STATS_H
 
 #include <grpc/support/port_platform.h>
+#include <grpc/support/string_util.h>
```

Comment:
Please move this include back to its original location, to comply with style guide rules:

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

(The only reason that port_platform.h is included at the top is that it is necessary in gRPC for portability reasons.)

Reply:
Woops, sorry, that was Vim being over-zealous. Done.